webui/react/src/e2e/models/ant/Select.ts

```diff
@@ -55,10 +55,20 @@ export class Select extends BaseComponent {
    */
   async openMenu(): Promise<Select> {
     if (await this._menu.pwLocator.isVisible()) {
-      return this;
+      // Try to close if still open. It might not be the right dropdown if another step left it open.
+      const locators = await this._menu.pwLocator.all();
```

Comment:
doublecheck make sure we need `.all`

Reply:
double checked and removed the loop, but kept the try/catch.